src/lib.rs

```diff
@@ -88,6 +132,10 @@ impl PrivateKey {
             .map(PrivateKey)
             .map_err(|_| JsValue::from_str("Invalid normal secret key"))
     }
+
+    pub fn sign(&self, message: &[u8]) -> Ed25519Signature {
+        Ed25519Signature(self.0.sign(&message.to_vec()))
```

Comment:
If there is a concern about people misusing this API, it may be better to explicitly require a TAG argument that way people know to use a unique prefix to avoid collision with other signable types